201-application-gateway-url-path-based-routing/azuredeploy.parameters.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@puneetsaraswat Can you update the parameters file to add the schema and parmeters sections? See here https://github.com/Azure/azure-quickstart-templates/blob/master/101-azure-dns-new-zone/azuredeploy.parameters.json